packages/public/server/__tests-e2e__/login.ts

```diff
@@ -19,49 +19,129 @@
  * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
  * @link      https://github.com/serlo-org/serlo.org for the canonical source repository
  */
-import { pages, viewports, users, elements } from './_config'
+import { pages, viewports, elements, testingServerUrl } from './_config'
 import {
   clickForNewPage,
   getByPlaceholderText,
   getBySelector,
   getByText,
+  getByRole,
   goto,
 } from './_utils'
+import { ElementHandle } from 'puppeteer'
 
-const examplePages = ['/', '/math', '/math/geometry/triangles']
+describe('login process with correct data', () => {
+  test('correct password and correct user', async () => {
```

Comment:
```suggestion
test('login process with correct data', async () => {
```

There is only one test in the describe-block -> thus it can be removed...